src/core/services/theming/theming.js

```diff
@@ -282,6 +357,8 @@ function ThemingProvider($mdColorPalette) {
       alwaysWatchTheme = alwaysWatch;
     },
 
+    enableBrowserColor: enableBrowserColor,
+
```

Comment:
Are you talking about [this](https://github.com/angular/material/pull/9192/files#diff-2a98812fedf7173f142f890cb0c29b85R544) or [that](https://github.com/angular/material/pull/9192/files#diff-2a98812fedf7173f142f890cb0c29b85R278)?